Dynatrace/dynatrace-operator
# Add annotation to exclude containers from injection.

## Description

Allow to exclude containers from injection based on their names with annotations on Dynakube or pod level.

## How can this be tested?

Deploy a Dynakube and an application.
Add annotation `container.inject.dynatrace.com/<containername> = false` to exclude app containers from injection.
Check that code modules are not injected into containers.

## Checklist
